tutorials/README.md

```diff
@@ -0,0 +1,29 @@
+# Tutorials
```

Comment:
#Use level 2 header `##` and make a single top-level header called something like `# Olin Electric Motorsports Monorepo` or something.